src/test/java/org/mockitousage/spies/MockCreationShouldNotAffectSpyTest.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockitousage.spies;
-
-import org.junit.Test;
-import org.mockitoutil.TestBase;
-
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-
-public class MockCreationShouldNotAffectSpyTest extends TestBase {
```

Comment:
Can we still keep these tests? I think they are valuable, especially when we attempt to reland the fix at a later point in time.